docs/user-guides/chain-builder.md

```diff
@@ -265,14 +265,17 @@ Now the wizard will output `partner-chains-public-keys.json` containing three ke
 
 ### 3. Run the prepare-configuration wizard
 
-Before running this wizard, be sure that `cardano-cli` is available and has an exposed socket to a running `cardano-node`.
+Before running this wizard, be sure that `ogmios` is available by host and port.
 
 1. Start the wizard:`./partner-chains-cli prepare-configuration`
 2. Update the bootnodes array and provide public ip or hostname
-3. Set the partner-chains parameters
-4. Store the main chain configuration
+3. Provide genesis utxo and required payment keys to initialize the chain
```

Comment:
I think the order is:
1) provide a payment key
2) then select genesis utxo from this key wallet